R/ObjectiveONNX.R

```diff
@@ -71,6 +84,8 @@ ObjectiveONNX = R6Class("ObjectiveONNX",
       self$retrafo = retrafo
 
       fun = function(xdt) {
+        # remove all-missing cols
+        xdt[, (which(colSums(is.na(xdt)) != 0)) := NULL]
```

Comment:
since recently this seems to generate a warning when there is no column to be removed.